app/src/main/java/com/example/whatshouldiread/customclasses/Question.java

```diff
@@ -0,0 +1,78 @@
+package com.example.whatshouldiread.customclasses;
+
+/*
+ * Class to store a question in text form as well as what position the question will be. Allows for
+ * multiple questions for the same position without changing the end result of the books.
+ */
+
```

Comment:
@petromw Updated the Javadocs for the files based on the comments from the original pull request. Please resolve this comment when you're okay with these changes.